docker-compose.yml

```diff
@@ -0,0 +1,19 @@
+version: '3'
+
+services:
+  pizza-sync-api:
+    build:
+      context: .
+      dockerfile: Dockerfile.api
+    image: pizza-sync-api
```

Comment:
I think you should add a simpler docker compose file that references only the image from docker hub for the users. It should be an example in the README, not a file in the repository.

The purpose of having this file in the repository would be for developpers like you or me, not users.
By the way, in this file, is that the official name of the image on docker hub?
If yes you should be able to use compose to push images when you build them...

Reply:
I don't know about that. I'm willing to discuss it but for people working on the project and willing to build the image with the current code, not the one on dockerhub... It's weird, isn't it?

I've already pushed to dockerhub btw: https://hub.docker.com/u/pizzasync/dashboard/

Reply:
that's why I'm saying:
 - for devs, you keep this compose file and use the same image name than the deployed image (so that you can build and push with compose).
 - for users you provide a compose file in the README as an example that refers to the hub's image so that users do not have to clone de repo if they want to run pizza-sync.

Reply:
btw this link to the hub does not work :)

Reply:
Oh sure thing I understand thanks.

I've updated the link (I just had to remove /dashboard, thanks for letting me know)